lab_6_pipeline/pipeline.py

```diff
@@ -112,11 +191,22 @@ def convert_morphological_tags(self, tags: str) -> str:  # type: ignore
         """
         Converts the Mystem tags into the UD format
         """
+        tags = re.sub(r'\((.+?)\|.+\)', r'\1', tags)
+        extracted_tags = re.findall(r'[а-я]+', tags)
+        ud_tags = {}
+        for tag in extracted_tags:
+            for category in (self.case, self.number, self.gender, self.animacy, self.tense):
```

Comment:
conceptually this is overwork, for example for nouns you look for tense and so on. You'd better define some more strict rules on what features you extract for each pos

Reply:
fixed